scripts/spec-pr.py

```
@@ -0,0 +1,135 @@
+    content = base64.b64encode(text.encode()).decode()
+    st, blob = call(token, "POST", f"/repos/{a.repo}/git/blobs",
+                    {"content": content, "encoding": "base64"})
+    if st != 201:
+        print(f"FATAL: blob 失败 {st} {blob}", file=sys.stderr)
+        sys.exit(2)
+    st, tree = call(token, "POST", f"/repos/{a.repo}/git/trees",
+                    {"base_tree": base_tree,
+                     "tree": [{"path": remote_path, "mode": "100644",
+                               "type": "blob", "sha": blob["sha"]}]})
+    st, commit = call(token, "POST", f"/repos/{a.repo}/git/commits",
+                      {"message": f"spec({taskId}): 条款级规格（auto，{a.ir_ref}，ADR-0050）",
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)